buildkite/benchmark/utils.sh

```diff
@@ -55,6 +55,20 @@ create_conda_env_for_benchmarkable_repo_commit() {
   conda activate "${BENCHMARKABLE_TYPE}"
 }
 
+create_conda_env_for_arrow_rs_commit() {
+  conda create -y -n "${BENCHMARKABLE_TYPE}" -c conda-forge \
+    python="3.9" \
+    rust
```

Comment:
Neat! I didn't know conda spoke Rust. I assumed it was Python only. Thanks for teach me this @ElenaHenderson!